apps/web-app/components/convos/upload.vue

```diff
@@ -0,0 +1,118 @@
+<script setup lang="ts">
+  /** Handles the file upload process for convo attachments.
+ * example usage:
+ * <ConvosUpload
+      v-model:uploadedAttachments="attachments"
+      :org-slug="orgSlug">
+      <template #default="{ openFileDialog, loading }">
+        <UnUiButton
+          :loading="loading"
+          label="refresh"
+          @click="openFileDialog" />
+      </template>
+    </ConvosUpload>
+ */
```

Comment:
Do we want to have the comment here?

Reply:
its pretty different usage than normal
i was hoping it would show in tooltips, but dosnt seem to work
where else should we document this?